src/main/java/de/orchound/gameoflife/parsing/RleBoardParser.java

```diff
@@ -0,0 +1,60 @@
+package de.orchound.gameoflife.parsing;
+
+import de.orchound.gameoflife.game.Board;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class RleBoardParser implements BoardParser {
+
+	private Board board;
+
+	@Override
+	public Board parse(File file) {
```

Comment:
Since you are merely converting the `File` object to a `Path`, I suggest using `Path` here already.
```suggestion
	public Board parse(Path file) {
```